src/frontend/main.go

```diff
@@ -186,7 +224,24 @@ func initTracing(log logrus.FieldLogger) {
 	// In a production environment or high QPS setup please use
 	// trace.ProbabilitySampler set at the desired probability.
 	trace.ApplyConfig(trace.Config{DefaultSampler: trace.AlwaysSample()})
-	initStackdriverTracing(log)
+	initOpenCensus(log)
+
+	// Initialize OpenTelemetry Tracer
+	projectID := os.Getenv("GOOGLE_CLOUD_PROJECT")
+	exporter, err := texporter.NewExporter(texporter.WithProjectID(projectID))
+	if err != nil {
+		log.Fatalf("failed to initialize exporter: %v", err)
+	}
+
+	// Create trace provider with the exporter.
+	tp, err := sdktrace.NewProvider(sdktrace.WithConfig(
+		sdktrace.Config{DefaultSampler: sdktrace.AlwaysSample()}),
+		sdktrace.WithSyncer(exporter),
+		sdktrace.WithResource(resource.New(standard.ServiceNameKey.String("GKE"))))
```

Comment:
Do you know if they support resource auto-detection? Or at least a way to pass a pre-defined constant for GKE?
If both are not an option, at least please add a comment with TODO so when resource autodetection is supported, we should fix it here.

Reply:
added a TODO comment
there's no no predefined constant in the `standard` api
I'm 90% sure no auto-detection

Reply:
u were right I didn't need it :P